dts/arm/st/l0/stm32l053.dtsi

```diff
@@ -4,23 +4,10 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <st/l0/stm32l0.dtsi>
+#include <st/l0/stm32l051.dtsi>
 
 / {
 	soc {
-		i2c2: i2c@40005800 {
-			compatible = "st,stm32-i2c-v2";
-			clock-frequency = <I2C_BITRATE_STANDARD>;
-			#address-cells = <1>;
-			#size-cells = <0>;
-			reg = <0x40005800 0x400>;
-			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00400000>;
-			interrupts = <24 0>;
-			interrupt-names = "combined";
-			status = "disabled";
-			label= "I2C_2";
-		};
-
 		spi2: spi@40003800 {
```

Comment:
spi2 should be moved in stm32l051

Reply:
Ok, moved

Reply:
Maybe I should change the name of the stm32l051.dtsi file to stm32l05x.dtsi?